examples/customizable/public/main.js

```diff
@@ -0,0 +1,124 @@
+import './carbon.min.js'
+import './browser.js'
+import './controls.js'
+import './footer.js'
+
+const updateForm = document.getElementById('update-form'),
+	updateEl = document.getElementById('update'),
+	customRefreshInput = document.getElementById('custom-refresh'),
+	disableOnBlurInput = document.getElementById('disable-on-blur'),
+	effectInput = document.getElementById('effect'),
+	colorInput = document.getElementById('color'),
+	appThemeInput = document.getElementById('app-theme'),
+	outOfSyncAction = document.getElementById('out-of-sync-action'),
+	outOfSyncNotification = document.getElementById('out-of-sync-notification')
+
+let color,
+	colorInvalid,
+	updateAllowed
+
+function toggleAllowUpdate(toggle = true) {
+	updateAllowed = toggle
+	updateEl.disabled = !updateAllowed
+}
```

Comment:
This function is not invoked except if you change the color. Please call this function in other cases, such as changing the customwindowrefresh option.

Reply:
I only disable the button if one of the values are invalid. The only value who can be invalid is color.

Reply:
I think the button is always enabled if the color is not invalid.

Reply:
If so, we need to revise the other part. In the current code, the button is disabled on the first run.

Reply:
I did not notice that. The initial state of the button would be to allow then?

Reply:
I think it is right to be allowed. I understood that the button is only activated when there is a modification because the button was not allowed in the initial state.

Reply:
I removed the "out-of-sync" actions, this also simplified the code _a lot_.